From 3f0c0a2fb522ee42509e752be7a5af2d6206f293 Mon Sep 17 00:00:00 2001
From: Brian Bennett <brian.bennett@joyent.com>
Date: Thu, 16 Nov 2017 16:11:13 -0800
Subject: [PATCH] joyent/manta-thoth#154 Need license info in package.json
 Reviewed by: Bryan Cantrill <bryan@joyent.com> Approved by: Bryan Cantrill
 <bryan@joyent.com>

---
 package.json | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package.json b/package.json
index 3fcb1fc..54eb376 100644
--- a/package.json
+++ b/package.json
@@ -2,6 +2,7 @@
 	"name": "manta-thoth",
 	"description": "Manta-based system for core and crash dump analysis",
 	"version": "1.0.0",
+    "license": "MIT",
 	"author": "Joyent (joyent.com)",
 	"dependencies": {
 		"manta": "^4.0.0",
-- 
2.21.0

